### PR TITLE
Provided parallel execution fix using asyncio for rbd trash purge test

### PR DESCRIPTION
**Pull request by sunilangadi2**
# Description
Fixes [RHCEPHQE-10043](https://issues.redhat.com//browse/RHCEPHQE-10043)
Failure logs:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-BZX7MW/

`fio `command is failing on one of the images, bcz before creating image fio is started executing in a parallel thread
This is causing the parallel loop to break with exception
also increased the sleep buffer to get updated results after all trash images got purged successfully.

Success logs:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-Q95H9W

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by sunilangadi2** [comment]
Latest changes success logs:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-NA2TSA

File: tests/rbd/test_rbd_trash_purge_schedule.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def verify_schedule_add(
     log.debug(f"Verifying purge on pool {pool}")
     while iterations_remaining > 0:
         log.debug(f"Sleeping for {schedule_interval} mins with buffer")
-        time.sleep(schedule_interval * 60 + 5)
+        time.sleep(schedule_interval * 60 + 60)
```
Comment (manasagowri) [inline tests/rbd/test_rbd_trash_purge_schedule.py:52]: Why did you increase the sleep time from 5 to 60?

**Review by manasagowri — COMMENTED** [review]
Just a small question

File: tests/rbd/test_rbd_trash_purge_schedule.py (hunk 1 above)
Comment (sunilangadi2) [reply tests/rbd/test_rbd_trash_purge_schedule.py:52]: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-T4XM6I/Test_to_verify_trash_purge_schedule_0.log
added buffer is not enough trash list still holds some entries after trash purge also.
hence after increasing sleep buffer issue not found

**Review by manasagowri — APPROVED** [review]
Changes LGTM

**Pull request merged**